sklearn/cluster/_affinity_propagation.py

```diff
@@ -72,6 +72,14 @@ def affinity_propagation(S, preference=None, convergence_iter=15, max_iter=200,
     return_n_iter : bool, default False
         Whether or not to return the number of iterations.
 
+    random_state : int or np.random.RandomStateInstance, default: None
+        Pseudo-random number generator to control the starting state.
+        Use an int for reproducible results across function calls.
+        See :term:`random_state`.
```

Comment:
I prefer something that says "see the Glossary" since "term" is not helpful to someone reading the text docs (rather than rendered HTML with links)